ios/PurchasesHybridCommon/PurchasesHybridCommon/CommonFunctionality.swift

```diff
@@ -8,8 +8,8 @@
 
 import Foundation
 import StoreKit
-import RevenueCat
 
+@testable import RevenueCat
```

Comment:
Wouldn't this cause compilation issues on `RELEASE` builds, where `RevenueCat` wouldn't be built for testing?

Reply:
Oh that's right this is not only in tests 🤦‍♂️